giskard/ml_worker/websocket/listener.py

```diff
@@ -156,13 +216,15 @@ def websocket_actor(action: MLWorkerAction):
     """
 
     def websocket_actor_callback(callback: callable):
-        if action in MLWorkerAction:
-            logger.debug(f'Registered "{callback.__name__}" for ML Worker "{action.name}"')
+        if action not in MLWorkerAction:
+            raise NotImplementedError(f"Missing implementation for {action}, not in MLWorkerAction")
+        logger.debug(f'Registered "{callback.__name__}" for ML Worker "{action.name}"')
+
+        def wrapped_callback(ml_worker: MLWorker, req: dict, *args, **kwargs):
+            dispatch_action(callback, ml_worker, action, req)
```

Comment:
in case `execute_in_pool` is added it would need to be passed here

Reply:
👍